extensions/erlang-mode/lsp-config.lisp

```diff
@@ -0,0 +1,25 @@
+(defpackage :lem-erlang-mode/lsp-config
+  (:use :cl :lem-lsp-mode :lem-lsp-base/type))
+
+(in-package :lem-erlang-mode/lsp-config)
+
+(defvar *lsp-erlang-elp-server-path*
+  (uiop:native-namestring "~/Applications/elp")) ; adapt to your own elp path
```

Comment:
you could place the comment as the defvar documentation.

(Is it really the default location?? Maybe add a runtime check that it exists?)